<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmMain</class>
 <widget class="QMainWindow" name="FmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GlyphWiki loader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edDirectory">
          <property name="text">
           <string notr="true">GlyphWikiRaw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Glyphs to load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="memoGlyphs">
        <property name="plainText">
         <string>31350
31353
31357
31359
3135C
31362
31367</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btStart">
          <property name="text">
           <string>LOAD!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btStop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbConsole">
        <property name="text">
         <string>Console</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="memoConsole">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
